docs/modules/ROOT/pages/backwards-compatibility.adoc

```diff
@@ -13,7 +13,9 @@ Infrequently a patch or minor update will remove or change an API in a breaking
 
 === Draft or Pre-Final ERCs
 
-ERCs that are not Final can change in incompatible ways. For this reason, we avoid shipping implementations of ERCs before they are Final. Some exceptions are made for ERCs that have been published for a long time and that seem unlikely to change. Breaking changes to the ERC specification are still technically possible in those cases, so these implementations are published in files named `draft-*.sol` to make that condition explicit.
+ERCs that are not Final can change in incompatible ways. For this reason, we avoid shipping implementations of ERCs before they are Final. Some exceptions are made for ERCs that have been published for a long time and seem unlikely to change. When breaking changes to the ERC specification remain technically possible, these implementations are published in files named `draft-*.sol` to make that condition explicit.
```

Comment:
I think this change is worth for clarifying when exceptions to the `draft-` prefix are made